website/content/commands/acl/translate-rules.mdx

```diff
@@ -24,13 +24,7 @@ are not supported from commands, but may be from the corresponding HTTP endpoint
 
 ### Usage
 
-Usage: `consul acl translate-rules [options] TRANSLATE`
-
-#### API Options
-
-@include 'http_api_options_client.mdx'
-
-@include 'http_api_options_server.mdx'
+Usage: `consul acl translate-rules [options] [TRANSLATE](#translate)`
```

Comment:
Note to self: check whether this anchor link works as expected

Reply:
Darn. I think links can only work if placed _outside_ the tickmarks. I'll have to take a different approach to this. (I think you may have suggested an alternative - need to finish looking at the review comments!)